dotcom-rendering/src/web/components/HeaderTopBarSearch.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { Search } from './HeaderTopBarSearch';
+
+export default {
+	component: Search,
+	title: 'Components/HeaderTopBarSearch',
+	parameters: {
+		backgrounds: { default: 'dark' },
+		layout: 'centered',
+	},
+};
+
+export const defaultStory = () => {
+	return <Search href="" dataLinkName="" />;
+};
+
+defaultStory.story = { name: 'Header top bar search' };
```

Comment:
I’m not entirely sure we gain much from having stories for the individual components of the top bar, especially as it’s an Island that’s hydrated as a whole.

I would rather we tested the possible variations of the whole header (edition, signed-in)